app/client/src/pages/Editor/PropertyPane/PropertyControl.tsx

```diff
@@ -794,15 +808,16 @@ const PropertyControl = memo((props: Props) => {
               tooltip={helpText}
             />
             {isConvertible && (
-              <Tooltip
-                content={JS_TOGGLE_DISABLED_MESSAGE}
-                isDisabled={!isToggleDisabled}
-              >
+              <Tooltip content={JSToggleTooltip} isDisabled={!JSToggleTooltip}>
                 <span>
                   <ToggleButton
-                    className={classNames("t--js-toggle", {
-                      "is-active": isDynamic,
-                    })}
+                    className={classNames(
+                      "t--js-toggle",
+                      {
+                        "is-active": isDynamic,
+                      },
+                      experimentalJSToggle && "!h-[20px]",
```

Comment:
```suggestion
                    className={classNames({
                        "t--js-toggle": true,
                        "is-active": isDynamic,
                        "!h-[20px]": experimentalJSToggle,
```
You should use the classNames to make this part cleaner. Please format the above correctly.